src/components/common/layout/header/index.tsx

```diff
@@ -1,14 +1,40 @@
+import Button from 'components/common/button';
+import SearchBar from 'components/common/search-bar';
+// import Image from 'components/common/image';
 import Image from 'next/image';
+import Link from 'next/link';
 
 const Header = () => (
-    <header className="fixed top-0 z-50 flex h-16 w-full items-center bg-black/25 px-4 backdrop-blur-xl backdrop-filter">
-        <Image
-            src="/logo.svg"
-            alt="Solana Logo"
-            layout="fixed"
-            width={175}
-            height={100}
-        />
+    <header className="flex flex-row justify-between fixed top-0 z-50 w-full h-20 items-center bg-black/25 px-6 backdrop-blur-xl backdrop-filter">
+        <div className="flex gap-10">
+            <Image
+                src="/logo.svg"
+                alt="logo"
+                width={175}
+                height={27}
+                // priority
+            />
```

Comment:
Is this where the custom `Image` didn't work as expected?

Reply:
Essentially yeah, I think the `width` and `height` `props` don't affeect `.svg`s for some reason. I also didn't know how to implement the `...rest` `prop` properly.

Reply:
I'll take a look at this.